projects/components/src/data-exporter/data-exporter.component.spec.ts

```diff
@@ -66,19 +66,11 @@ describe('DataExporterColumnsWithoutDisplayName', () => {
         const exporter = this.finder.find<DataExporterWidgetObject<TestElement>>(DataExporterWidgetObject);
         exporter.getToggleSelectAll().click();
         this.finder.detectChanges();
-        const dropdownMenuItemTexts = exporter.getColumnBubbles();
+        const columnBubbles = exporter.getColumnBubbles().map((i) => i.text());
+        const columnCheckboxLabels = exporter.getColumnCheckbox().map((i) => i.text());
         const expects = ['col1', 'col2'];
-        for (const item of dropdownMenuItemTexts) {
-            expect(item.text()).toEqual(expects.pop());
-        }
-        expect(dropdownMenuItemTexts.text()).toEqual('col1');
-
-        exporter.getColumnCheckbox(0).click();
-
-        const columnBubbles = exporter.getColumnBubbles();
-        exporter.getColumnDropdown().click();
-
-        expect(exporter.getColumnBubbles().text()).toEqual('col2');
+        expect(columnBubbles).toEqual(expects);
+        expect(columnCheckboxLabels).toEqual(expects);
```

Comment:
@ryan-bradford  I couldn't understand what this was doing previously, it seemed unrelated to what this test is for. Should I create another test?

Reply:
I think I patched this onto the test to get coverage for removing the column VIA the bubbles. Making another test would probably be good.

Reply:
Thanks, I'll add its own it for that. Let's avoid covering two problems in one unit test.